controllers/drpolicy_controller.go

```diff
@@ -166,16 +165,11 @@ func (r *DRPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 
 func (r *DRPolicyReconciler) createOrUpdateManifestWorkForVRC(ctx context.Context, mp *multiclusterv1alpha1.MirrorPeer, dp *ramenv1alpha1.DRPolicy, clusterFSIDs map[string]string) error {
 
-	var fsids []string
-	for _, v := range clusterFSIDs {
-		fsids = append(fsids, v)
+	replicationId, err := utils.CreateUniqueReplicationId(clusterFSIDs)
+	if err != nil {
+		return err
```

Comment:
Returning an error will keep it in continous reconcilation and will keep printing the logs.
Do we want to log the info and reconcile at a later interval ?

Reply:
Error loop is fine I guess, since we do not expect it to be for long. Also, better to error out that a manual restart.